package/versionconfig.ini

```diff
@@ -34,6 +34,7 @@ qres32.dll
 ls800obs.shp
 mplsobs.pal
 ra2mdlauncher.exe
+ra2md-launcher.sh
```

Comment:
is it better to make the file name consistent? I mean considering removing the hyphen

Reply:
> is it better to make the file name consistent? I mean considering removing the hyphen

I personally prefer having readable file names. 🤔